modules/openapi-generator/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -455,7 +455,12 @@ namespace {{packageName}}.Client
             var clientOptions = new RestClientOptions(baseUrl)
             {
                 ClientCertificates = configuration.ClientCertificates,
+                {{#useMaxTimeout}}
                 MaxTimeout = configuration.Timeout,
+                {{/useMaxTimeout}}
+                {{^useMaxTimeout}}
+                Timeout = TimeSpan.FromMilliseconds(configuration.Timeout),
```

Comment:
Open to making the configuration property in IReadableConfiguration a TimeSpan

Reply:
to me, it make senses and it's another breaking change with fallback so we should include it in this PR